parts/kubernetesmastercustomscript.sh

```diff
@@ -249,6 +249,23 @@ function ensureEtcd() {
     done
 }
 
+function ensureEtcdDataDir() {
+    etcdStarted=1
+    mount | grep /dev/sdc1 | grep /var/lib/etcddisk
+    if [ "$?" = "0" ]
+    then
+        echo "Etcd running with data dir at: /var/lib/etcddisk"
+        etcdStarted=0
+        break
+    fi
+
+    if [ $etcdStarted -ne 0 ]
+    then
+        echo "Etcd data dir was not found at: /var/lib/etcddisk"
```

Comment:
The retry I was imagining was something similar to what we have in ensureApiServer and ensureDocker with a sleep and call to "mount -a", that will just kick anything unmounted in fstab.  We can wait until you can determine reliability and decide then.  I'll approve now so you can get this in.